charts/jaeger/templates/collector-deploy.yaml

```diff
@@ -43,74 +43,33 @@ spec:
 {{ toYaml .Values.collector.securityContext | indent 10 }}
         image: {{ .Values.collector.image }}:{{ .Values.tag }}
         imagePullPolicy: {{ .Values.collector.pullPolicy }}
-        env:
-          {{- range $key, $value := .Values.collector.cmdlineParams }}
-          - name: {{ $key | replace "." "_" | replace "-" "_" | upper | quote }}
-            value: {{ $value | quote }}
+        args:
+          {{- range $key, $value := .Values.collector.cmdlineParams -}}
+          - --{{ $key }}={{ $value }}
```

Comment:
some command line params do not have a value, right? Maybe the `={{ $value }}` part should be optional

Reply:
I am having trouble finding an example of this: https://www.jaegertracing.io/docs/1.17/cli/#jaeger-collector

Could you point one out?

Reply:
@jpkrohling @yurishkuro Could you please confirm if any command line options for any of the components are in the form of `--key` without a value (i.e. `--key=value`) ?

Reply:
AFAIK, boolean flags only need a key...

Reply:
Alright, please see https://github.com/jaegertracing/helm-charts/pull/75/commits/8d36352e78b758e2fd7eb4d3006713a16a4a8366

Reply:
Voilà!

https://github.com/jaegertracing/helm-charts/pull/75/commits/64352aea620b02d8cb8c13ae786df842af8348c3